docs/content/guides/developer/app-examples/reviews-rating.mdx

```diff
@@ -0,0 +1,212 @@
+  User ->> Service: Send review
+  Service ->> User: Send reward
+  Service ->> Moderator: Add as a service moderator
+  Moderator ->> Service: Remove abused review
+```
+
+### Service owners
+
+Service owners are entities like restaurants that list their services on the platform. Service owners allocate a specific amount of SUI as a reward pool. Assets from the pool are used to provide rewards for high-rated reviews. A Proof of Experience (PoE) NFT confirms the reviewer used the service, which they burn to provide a verified review. Service owners can provide their customers with unique identifiers, perhaps using QR codes, to identify individual reviewers.
+
+### Reviewers
+
+Reviewers are consumers of services that use the review system. Reviewers provide feedback in the form of comments that detail specific aspects of the service as well as a star rating to inform others. The reviews are rated, with the most effective reviews getting the highest rating. Service owners award the 10 highest rated reviews for their service. How often the rewards are distributed is up to the service owner; it can be on a regular or rolling basis.
+
+### Review readers
+
+Review readers access reviews to make informed decisions on selecting services. Readers rate reviews by casting up or down votes. The review readers' ratings are factored into the algorithm that rates the reviews, with the authors of the highest-rated reviews getting awarded. 
+
+### Moderators
+
+Moderators monitor content of the reviews and can delete any reviews that contain inappropriate content.
+Moderators also keep an eye on the number of reviews registered for a service to ensure that they do not exceed the maximum number allowed.
+When the maximum number of reviews is reached, moderators can prune old reviews to make room for a new review.
+
+## How reviews are rated
+
+The reviews are rated (scored) on-chain using following criteria
+- Intrinsic score (IS)
+  - Length of content
+- Extrinsic score (ES)
+  - Number of votes received
+- Verification multiplier (VM)
+  - Reviews with PoE get a multiplier to improve rating
+
+```
+Total Score = (IS + ES) * VM
+```
+
+## Smart contracts
+
+There are several modules that create the backend logic for the example. 
+
+### dashboard.move
+
+The `dashboard.move` module defines the `Dashboard` struct that is used to group services.
+
+```move
+struct Dashboard has key, store {
+  id: UID,
+  set: VecSet<ID>,
+  service_type: String
+}
+```
+
+The services are grouped by attributes, which can be cuisine type, geographical location, operating hours, google maps ID, and so on. To keep it basic, the example stores only `service_type` (for example, fast food, Chinese, Italian).
+
+### service.move
+
+This module defines the `Services` struct, which service owners manage.
+
+```move
+struct Service has key, store {
+  id: UID,
+  reward_pool: Balance<SUI>,
+  reward: u64,
+  reviews: MultiMap<ID>,
+  moderators: VecSet<address>,
```

Comment:
If you are relying on moderators to identify when the service is spoofing reviews, there's a conflict of interest in the service specifying who its moderators are.

Reply:
I was thinking that service was managed by well known moderators (with well known sui address), and it was ok for service to publicly acknowledge that its moderator is who. It's kind of how buildings are managed by well known security company such as ADT which typically put huge sticker or signs near the entrance of the building.